package.json

```diff
@@ -101,6 +101,10 @@
       {
         "title": "dbt Power User",
         "properties": {
+          "dbt.unquotedCaseInsensitiveIdentifierRegex": {
```

Comment:
new config for users if regex [here](https://github.com/AltimateAI/vscode-dbt-power-user/pull/1117/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R211) does not work for a project